api/reporting/schemas/input/descriptions/user.yaml

```diff
@@ -0,0 +1,60 @@
+---
+user:
+  about_you: User's description of themselves.
+  area_of_research: 
+  compliance_training_bypass_time: |
+        Time Compliance Training administratively bypassed, or null if this requirement
+        was never bypassed for this user.
+  compliance_training_completion_time: |
+    Time Compliance Training completed, or null if never completed.
+  compliance_training_expiration_time: |
+      Time  User's compliance training will expire, or null training not taken or if bypassed
+  contact_email: User's external email address on file.
+  creation_time: Timestamp of account creation.
+  current_position: User's current role/position.
+  data_access_level: |
+    An indication of whether the user has completed the requirements for access to the
+    registered tier.
+  data_use_agreement_bypass_time: |
+    Time an admin bypassed the DUCC rerquirement for this user, or null if that has not happened.
+    This value is reset to null if a user's bypassed status  is revoked.
+  data_use_agreement_completion_time: |
+    Time user last completed DUCC
+  data_use_agreement_signed_version: |
+    Version of DUCC last signed by this user.
+  demographic_survey_completion_time: |
+    Timestamp for completion of Demographic Survey (if completed).
+  disabled: |
+    If true, this account has been disabled by an administrator (or potentially an automatic process).
+  era_commons_bypass_time: |
+    Time an administrator bypassed the ERA Commons requirement for this user.
+  era_commons_completion_time: |
+    Time user completed ERA Commons account link.
+  family_name: |
+    User last name (family name).
+  first_registration_completion_time: |
+    Time user first completed registration.
+  first_sign_in_time: |
+    Time user first signed into the Workbench with a GSuite account.
+  free_tier_credits_limit_days_override:
+    Override value for the default free tier time limit (days).
+  free_tier_credits_limit_dollars_override: |
+    Override value for the default free tier spending limit (USD).
+  given_name: |
+    User first name (given name).
+  last_modified_time: |
+    Time of last modification to this user account.
+  professional_url: |
+    User's URL at primary place of work.
+  two_factor_auth_bypass_time:  |
```

Comment:
nit: can we put all bypass_time elements and completion_time together?

Reply:
no, I need to  keep  it alphabetical  for  merging, tracking across  files, etc.